frontend/institution/institutionController.js

```diff
@@ -189,6 +189,11 @@
             $state.go('app.institution.events', {institutionKey: institutionKey, posts: institutionCtrl.posts});
         };
 
+        institutionCtrl.goToLinks = function goToLinks(institutionKey) {
```

Comment:
test it. Even this function being so small, the tests can protected it from changes.